bin/storer.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3.6
+
+import argparse, signal, sys
+from concurrent.futures import ProcessPoolExecutor
+from urllib3.util import parse_url, Url
+from bundle_tools import logger, StagedBundle, StagedBundleFinder, BundleStorer, DataStoreAPI
+
+"""
+    Store staged bundles in the HCA DSS Data Store
+    
+    Bundle must have been previously staged.
+    s3 URL of a bundle or bundle hierarchy (which has at least one 'bundles' subfolder) must be supplied, e.g.
+    
+    bin/storer.py --bundles s3://org-humancellatlas-data-bundle-examples/import/10x
+    
+    bin/storer.py --bundle s3://org-humancellatlas-data-bundle-examples/import/geo/GSE75478/bundles/bundle145
+"""
+
+
+# Executor complains if it is an object attribute, so we make it global.
+executor = None
+
+
+class Main:
+
+    DEFAULT_STAGED_BUNDLES_URL = "s3://org-humancellatlas-data-bundle-examples/import"
+
+    def __init__(self):
+        parser = argparse.ArgumentParser(description="Store staged bundles in HCA Data Store")
```

Comment:
Consider using `parser = argparse.ArgumentParser(description=__doc__)`. In order for that to work, you'd want to move the docblock above the imports, right after the shebang line.